files/en-us/web/api/htmlmapelement/name/index.html

```diff
@@ -0,0 +1,46 @@
+---
+title: "HTMLMapElement: name property"
+short-title: name
+slug: Web/API/HTMLMapElement/name
+page-type: web-api-instance-property
+browser-compat: api.HTMLMapElement.name
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`name`** property of the {{domxref("HTMLMapElement")}} is a string gives the map unique name. 
```

Comment:
```suggestion
The **`name`** property of the {{domxref("HTMLMapElement")}} represents the map's unique name. 
```

Reply:
```suggestion
The **`name`** property of the {{domxref("HTMLMapElement")}} is a string gives the `<map>` unique name. 
```
<map> element is better than map's